doc/quickstart.md

```diff
@@ -0,0 +1,92 @@
+## Deep Insights
+
+This document is a quick start to use CartoDB Deep Insights (DI). It shows how to create dashboards and modify widgets from a JavaScript application.
+
+**Important notice**
+This is a beta version, so the API interface might change. We will do our best effort to keep all the changes backwards compatible.
+
+### Using Deep insights library in your application
+
+As this is a WIP library, it will be updated daily with bug fixes, for that reason it is
+encouraged to use the CDN version of the library so as we update library your application gets the latests fixes.
+
+```htmlc
+<link rel="stylesheet" href="http://libs.cartocdn.com/di.js/v0/themes/css/deep-insights.css" />
+<script src=" http://libs.cartocdn.com/di.js/v0/deep-insights.uncompressed.js"></script>
+http://libs.cartocdn.com/di.js/v0/deep-insights.uncompressed.js
+```
+
+
+### Create a dashboard
+
+Here's how to create a dashboard:
+
+```js
+cartodb.deepInsights.createDashboard('#dashboard', vizJSONurl, function(err, dashboard) {
+});
+```
+
+## Found a bug?
+
+Please report it in [deep insights repo](https://github.com/CartoDB/deep-insights.js/issues)
+
+## API
+
+### cartodb.DI.createDashboard(domObject, vizJson, callback(err, obj))
+
+Creates a new dashboard inside `domObject` based on `vizJson`.
+
+### cartodb.DI.Dashborbard
+
+This object contains a CartoDB dashboard, a map, and some widgets:
+
+#### getWidgets() -> Array of widgets
+#### getWidget(id) -> get widget
+#### getMap() -> returns a Map object
+
+#### createCategory-> CategoryWidget
```

Comment:
IMO `createCategoryWidget`, `createHistogramWidget`, etc. would be more descriptive.